packages/mira/tasks/generate_model_latex.py

```diff
@@ -0,0 +1,76 @@
+import sys
+import json
+import traceback
+from taskrunner import TaskRunnerInterface
+import sympy
+from mira.sources.amr import model_from_json
+
+def cleanup():
+    pass
+
+def main():
+    exitCode = 0
+
+    try:
+        taskrunner = TaskRunnerInterface(description="Generate latex")
+        taskrunner.on_cancellation(cleanup)
+
+        data = taskrunner.read_input_str_with_timeout()
+        amr = json.loads(data)
+        model = model_from_json(amr)
+
+        odeterms = {var: 0 for var in model.get_concepts_name_map().keys()}
+
+        for t in model.templates:
+            if hasattr(t, "subject"):
+                var = t.subject.name
+                odeterms[var] -= t.rate_law.args[0]
+
+            if hasattr(t, "outcome"):
+                var = t.outcome.name
+                odeterms[var] += t.rate_law.args[0]
+
+        # Time
+        symb = lambda x: sympy.Symbol(x)
+        try:
+            time = model.time.name
+        except:
+            time = "t"
+        finally:
+            t = symb(time)
```

Comment:
This try/except block feels a bit lazy - can we do proper checks to see if time is valid rather than letting it blow up and fall into except/final blocks?

Reply:
Lines 35-39 are probably not necessary. The sympy equations just need a time symbol (which is defined in Line 40).

Reply:
Yep I will fix this